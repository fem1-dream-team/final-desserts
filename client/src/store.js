import {createStore, applyMiddleware} from 'redux';
// import thunk from 'redux-thunk';
import createSagaMiddleware from 'redux-saga'

import rootReducer from './reducers';
import {rootSaga} from './actions/sagaAuthForm'
// const initialState = {};
// const thunkMiddleware = [thunk];

const saga = createSagaMiddleware()

const store = createStore(rootReducer, applyMiddleware(saga));
saga.run(rootSaga)

export default store