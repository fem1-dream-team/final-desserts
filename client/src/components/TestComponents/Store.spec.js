import store from '../../store'
import React from "react";
import renderer from 'react-test-renderer'

describe("store component", () => {

	it("store component render", () => {
		const wrapper = renderer.create(<store />).toJSON();
		expect(wrapper).toMatchSnapshot();

	});
});