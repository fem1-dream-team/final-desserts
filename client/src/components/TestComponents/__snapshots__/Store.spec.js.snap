// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store component store component render 1`] = `<store />`;
